src/system/controlsystem.jl

```diff
@@ -1,3 +1,5 @@
+using IntervalArithmetic
```

Comment:
```suggestion
import IntervalArithmetic
```

Reply:
You can also do `import ... as IA` if it is used at many places

Reply:
I clarify it most of the places... I will deal with the remaining by refactoring the concerned solvers..